SlimApp/UI/NewClient/NewClientView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  NewClientView.swift
+//  SlimApp
+//
+//  Created by Ekaterina Grishina on 11/01/23.
+//
+
+import SwiftUI
+
+struct NewClientView: View {
+    
+    @StateObject var viewModel: NewClientViewModel
+    
+    var body: some View {
+        NavigationView {
+            // TODO: e.grishina rewrite it with ScrollView
+            Form {
+                Section {
+                    TextField("Name", text: $viewModel.name)
+                    TextField("Surname", text: $viewModel.surname)
+                    TextField("E-mail", text: $viewModel.email)
+                        .keyboardType(.emailAddress)
+                    TextField("Phone number", text: $viewModel.phoneNumber)
+                        .keyboardType(.phonePad)
+                }
+                Section {
+                    TextField("Height", text: $viewModel.height)
+                        .keyboardType(.decimalPad)
+                    TextField("Weight", text: $viewModel.weight)
+                        .keyboardType(.numberPad)
+                }
+                Section {
+                    DayPicker(days: $viewModel.workoutDays)
+                        .buttonStyle(.borderless)
+                        .listRowInsets(EdgeInsets(top: 4, leading: 4, bottom: 4, trailing: 4))
+                }
+                Section("Notes") {
+                    TextEditor(text: $viewModel.notes)
+                }
+                Button {
+                    #warning("Implement saving client information")
+                } label: {
+                    Text("Save")
+                        .frame(width: UIScreen.main.bounds.width)
+                }
+                .buttonStyle(.borderless)
+            }
+            .navigationTitle("New client")
```

Comment:
Maybe here we could "force" the title as a '.inline' to recover some space in the view.

Reply:
Large title looks better here, imho. We can discuss